### PR TITLE
Feature: Removed fluid interface from module entity/interface

### PR DESCRIPTION
**Pull request by ins0**
- added missing doctypes to the module interface
- removes return types from interface
- removes fluid style from module entity

### CODE REVIEW

File: module/ZfModule/src/ZfModule/Entity/ModuleInterface.php [hunk 1]
```diff
@@ -6,12 +6,16 @@
...
+     *
      * @return string
      */
     public function getOwner();
 
     /**
      * Set Owner
+     *
+     * @param string
+     * @return string
```
Comment (localheinz) [inline module/ZfModule/src/ZfModule/Entity/ModuleInterface.php]: `@return self`?

Or maybe even nothing? Not sure, but I remember @Ocramius having some arguments against fluent interfaces some time ago!

Comment (ins0) [inline module/ZfModule/src/ZfModule/Entity/ModuleInterface.php]: you're right :+1: for removing the fluid interface

Comment (localheinz) [inline module/ZfModule/src/ZfModule/Entity/ModuleInterface.php]: @ins0 

For reference, here's @Ocramius blog post: http://ocramius.github.io/blog/fluent-interfaces-are-evil/

Comment (ins0) [inline module/ZfModule/src/ZfModule/Entity/ModuleInterface.php]: yeah was reading this after it was published - thanks. may should i extend the pr to remove the fluid style?

Comment (localheinz) [inline module/ZfModule/src/ZfModule/Entity/ModuleInterface.php]: I think it's just ok to remove the `@return` from the interface, probably!

**Comment by ins0** [comment]
@Ocramius sry this switched from a Fix to a Feature :>

**Pull request merged**